### DISCUSSION TITLE
Where did all the time (host minutes) go?

### DISCUSSION THREAD

**Discussion by andy-brainome** [community/community#27032] [Actions]
<p>I’m still trying to figure out where did all of the actions minutes go. My last post gave me billable charges but somehow my 3000 included minutes are not accounted for.<br>
My issue is that a seemingly small 1 hour total runtime job has totally blown through nearly 4000 minutes in less that two days.<br>
Yes I know now that I should have put on timeout guard rails to prevent this from bankrupting my budget but I still don’t have a complete accounting for this situation.</p>
<p>so - a) how do I get a report for ALL the TIME attributed to my account?<br>
b) how do I recover the credits taken away from me due to a github systems failure where the same task ran through to completion in less than 15 minutes every other time?</p>

**Comment by ghost** [comment]
<p>Hi, sorry to hear about this.</p>
<p>Firstly, we recommend downloading a usage report:<br>
<a href="https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage</a></p>
<p>Please note the usage report is in UTC, but the Actions web UI is in your local configuration time settings. So you may need to adjust your local configuration to UTC to line up the usage report.</p>
<p>You can learn more about how we calculate minutes here:<br>
<a href="https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions</a></p>
<p>As for b), I believe you can open a support ticket by choosing <code>Account or billing issues</code> here:</p>
<p><a href="https://support.github.com/request" class="onebox" target="_blank" rel="noopener">https://support.github.com/request</a></p>

**Comment by andy-brainome** [comment, marked as answer]
<p>The crazy thing is that the macos minutes are charged at 10x linux minutes.</p>
